### PR TITLE
Update min SDK version to `2.15.0`

### PR DESCRIPTION
**Pull request by elliette**
I'm seeing errors in https://github.com/dart-lang/webdev/pull/1462 about the min SDK version. 

Updating the SDK version here instead of in that PR.

### CODE REVIEW

File: webdev/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2.7.7
```
Comment (annagrin) [inline webdev/CHANGELOG.md:1]: Lets keep the version of webdev unchanged, we will need to change it to update dwds and publish later

**Review by annagrin — CHANGES_REQUESTED** [review]
Thanks you Elliott! Let's keep webdev version the same for now, otherwise LGTM!

File: webdev/CHANGELOG.md (hunk 1 above)
Comment (elliette) [reply webdev/CHANGELOG.md]: Reverted in back to 2.7.7-dev

File: dwds/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 11.5.1-dev
```
Comment (annagrin) [inline dwds/CHANGELOG.md:1]: Sorry, forgot to mention - lets just make it 11.5.1 so we can prepare to publish

**Review by annagrin — CHANGES_REQUESTED** [review]

File: dwds/CHANGELOG.md (hunk 2 above)
Comment (elliette) [reply dwds/CHANGELOG.md]: Done!

**Review by annagrin — APPROVED** [review]
LGTM!

**Pull request merged**